cluster/charts/rook-ceph-cluster/templates/ingress.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.ingress.host }}
+---
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
+apiVersion: networking.k8s.io/v1beta1
+{{ else }}
+apiVersion: extensions/v1beta1
+{{ end -}}
+kind: Ingress
+metadata:
+  name: {{ template "clusterName" . }}-dashboard
+  {{- if .Values.ingress.annotations }}
+  annotations: {{- toYaml .Values.ingress.annotations | nindent 4 }}
+  {{- end }}
+spec:
+  rules:
+    - host: {{ .Values.ingress.host.name }}
+      http:
+        paths:
+          - path: {{ .Values.ingress.host.path }}
+            backend:
+              serviceName: rook-ceph-mgr-dashboard
+              servicePort: http-dashboard
```

Comment:
Since ssl termination is handled by the ingress, how about a comment that `dashboard.sslEnabled: true` is not supported?

Reply:
> ssl: Whether to serve the dashboard via SSL, ignored on Ceph versions older than 13.2.2

from https://github.com/rook/rook/blob/master/Documentation/ceph-cluster-crd.md

hmm, I guess that isn't being used, I've only ever used ceph since v13.2.5 so never noticed that ever making an impact. I'll remove it from this PR, should I create an issue to remove that piece of configuration from documentation because rook doesn't support v13 mimic anymore

Reply:
Yeah looks like have some old references to clean up. Nautilus is the oldest currently supported